polygon/rest/reference.py

```diff
@@ -106,6 +107,7 @@ def list_tickers(
         :param sort: The field to sort the results on. Default is ticker. If the search query parameter is present, sort is ignored and results are ordered by relevance.
         :param order: The order to sort the results on. Default is asc (ascending).
         :param params: Any additional query params.
+        :param headers: X-Edge Headers for Launchpad partners.
```

Comment:
There's gotta be a way to include the headers for every possible endpoint without adding it to each individual function call.